server/src/app.ts

```diff
@@ -19,6 +20,21 @@ mongoose.connect(process.env.MONGODB_URL || '', options)
   .catch(e => console.error(e));
 
 const app = express();
+app.use(passport.initialize());
+localStrategy();
+
+const authenticationResolver = ['loginQuery', 'signupQuery'];
+const server = new ApolloServer({ typeDefs, resolvers, context: ({ req, res }) => {
+  console.log(req.body);
+  if (authenticationResolver.includes(req.body.operationName)) {
+    console.log('login');
```

Comment:
```suggestion
  if (authenticationResolver.includes(req.body.operationName)) {
```
console.log()가 단순 확인 용도라면 없어도 될 것 같습니다.

Reply:
넵 이것도 놓칠뻔 했네용 수정해놓을게요 감사합니다!